services/queue/src/api.js

```diff
@@ -775,29 +752,12 @@ builder.declare({
   scopes: {AllOf: [
     {for: 'scope', in: 'scopes', each: '<scope>'},
     {for: 'route', in: 'routes', each: 'queue:route:<route>'},
+    'queue:scheduler-id:<schedulerId>',
     {AnyOf: [
-      {AllOf: [
-        'queue:scheduler-id:<schedulerId>',
-        {AnyOf: [
-          {
-            for: 'priority',
-            in: 'priorities',
-            each: 'queue:create-task:<priority>:<provisionerId>/<workerType>',
-          },
-        ]},
-      ]},
       {
-        if: 'legacyScopes',
-        then: {AnyOf: [
-          'queue:define-task:<provisionerId>/<workerType>',
-          'queue:create-task:<provisionerId>/<workerType>',
-          {
-            AllOf: [
-              'queue:define-task:<provisionerId>/<workerType>',
-              'queue:task-group-id:<schedulerId>/<taskGroupId>',
-            ],
-          },
-        ]},
+        for: 'priority',
+        in: 'priorities',
+        each: 'queue:create-task:<priority>:<provisionerId>/<workerType>',
```

Comment:
While we're poking around in here are we ready to do bug 1455632 at this point? It would be kinda neat if we enforced `APIBuilder.stability.deprecated` needing a bug number associated with it somehow.

Reply:
I suspect that yes, we could remove defineTask as well, which would close that bug up (we want to keep rerunTask).

And yeah, that sounds like a good contributor project.

Reply:
Filed https://bugzilla.mozilla.org/show_bug.cgi?id=1619808